src/resources/how_to/how_to_manage_api_keys_as_project_resources.md

```diff
@@ -0,0 +1,111 @@
+# How to Manage API Keys as Project Resources
+
+API keys in Aptos Build provide authenticated access to enhanced services with higher rate limits and additional features. This guide covers Full Node API keys and references to other API key types.
+
+## Prerequisites
+
+- Active Aptos Build account at [build.aptoslabs.com](https://build.aptoslabs.com)
+- Node.js project with package.json
+- Basic understanding of environment variables
+
+## Full Node API Keys
```

Comment:
I would actually pull the full node api key section to a separate resource file, and leave this file as an overview of all the api keys Aptos supports.
Then the agent can reference to a specific resource based on the user prompt